src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -132,6 +138,14 @@ public TileChest()
 
 		this.setInternalPublicPowerStorage( true );
 		this.setInternalPowerFlow( AccessRestriction.WRITE );
+
+		this.inputInventory.setFilter( new InputInventoryFilter() );
+		this.cellInventory.setFilter( new CellInventoryFilter() );
+	}
+
+	public ItemStack GetCell()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00100)